.github/workflows/ci.yml

```diff
@@ -1884,22 +1886,22 @@ jobs:
       - id: check
         run: |
           set +e
-          ./scripts/is_version_already_uploaded.sh cargo-semver-checks
+          "./scripts/is_version_already_uploaded.sh" "cargo-semver-checks"
           export EXIT_CODE="$?"
           set -e
           if [[ "$EXIT_CODE" == "7" ]]; then
-            echo 'is_new_version=no' >> $GITHUB_OUTPUT
+            echo "is_new_version=no" >> "$GITHUB_OUTPUT"
           elif [[ "$EXIT_CODE" == "0" ]]; then
-            echo 'is_new_version=yes' >> $GITHUB_OUTPUT
+            echo "is_new_version=yes" >> "$GITHUB_OUTPUT"
```

Comment:
These changes are strange too. I agree with `"$GITHUB_OUTPUT"` but everything else seems _at best_ a preference and at worst making the code less good.